client/app/instructor/events/events.html

```diff
@@ -19,7 +19,14 @@ <h1 class="page-header">Events</h1>
                 </div>
               </div>
               <div>
-                {{event.info.description}}
+                {{event.description}}
+              </div>
+              <div class="event-section">
+                <ul>
+                  <li ng-repeat="sectionEvent in event.sections" ng-click='goToEvent(sectionEvent)'>
+                    <b>Course:</b> {{sectionEvent.course}}, <b>{{(sectionEvent.sectionNumbers.length===1)?"Section: ":"Sections: "}}</b> <span ng-repeat="num in sectionEvent.sectionNumbers"> {{num}}<span ng-hide="$last">, </span></span>
```

Comment:
Similar to above (but I like this one better!) Just some formatting changes perhaps @samanthajlee2 ? (The instructor events page)